data/openings.js

```diff
@@ -0,0 +1,23 @@
+export const openings = [
+
+
+    {
+        id: 1,
+        title: "Business Analyst",
+
+    },
+    {
+        id: 2,
+        title: "Frontend Engineer",
+
+    },
+    {
+        id: 3,
+        title: "Python Developer",
+    },
+    {
+        id: 4,
+        title: "Senior Project Manager ",
+    },
+
+]
```

Comment:
You should have the same coding style while declaring and exporting files and components.
Try also to have the same prettier configuration so that the code can be formatted in one style.